generator_hdl.py

```diff
@@ -290,16 +290,27 @@ def writeTBMemoryWrites(memDict, memInfoDict, notfinal_procs,split, MPARdict):
 
         if memInfo.isFIFO:
             string_tmp = writeTBMemoryWriteFIFOInstance(mtypeB, memDict, proc)
-            # A bodge for TrackBuilder to write TF concatenated track+stub data.
+            # Code for TrackBuilder to write TF concatenated track+stub data.
             # (Needed to compare with emData/).
-            if mtypeB == 'TW_104':
+            if mtypeB == 'TW_113':
```

Comment:
@aryd The magic 113 can be removed with something like the following:
```suggestion
            if mtypeB.startswith('TW_'):
```